### PR TITLE
make read only workspaces (blockly/applab) visually distinct

### PR DESCRIPTION
**Pull request by laurelfan**
Before:

(looks exactly the same as the editable workspace)

After:

### CODE REVIEW

File: apps/package.json [hunk 1]
```diff
@@ -48,7 +48,7 @@
     "grunt-lodash": "~0.3.0",
     "grunt-newer": "^1.1.0",
     "grunt-notify": "^0.4.1",
-    "grunt-sass": "~0.17.0",
+    "grunt-sass": "~1.0.0",
```
Comment (laurelfan) [inline apps/package.json]: I had to change this to make my build work with npm 2.10.1

**Comment by sfilman** [comment]
Looks great!

**Comment by Bjvanminnen** [comment]
When do we have a readonly workspace (other than in feedback dialogs). Should also probably make sense that feedback dialogs look okay with this change.

**Comment by laurelfan** [comment]
This is used when we want to display the full workspace but not allow editing (the not-editing feature already exists, this just makes it look different from the editable workspace), which happens in these cases:
* teacher viewing the answer for a puzzle
* teacher viewing a student's solution
* viewing a /p/ project /edit url if you are not the project owner

feedback dialogs are not affected:



because it's a different code path (StudioApp is initialized with initReadonly() instead of init() with config.readonlyWorkspace) and different css (.readonly #codeWorkspace vs #codeWorkspace.readonly)

**Comment by Bjvanminnen** [comment]
lgtm

**Pull request merged**